content/blog/2018-07-24-educollab-challenges.md

```diff
@@ -0,0 +1,44 @@
+---
+slug: educollab-challenges
+title: "rOpenSci Educators Collaborative: What Are The Challenges When Teaching Science With R?"
+date: 2018-07-24
+preface: "At the [5th annual rOpenSci unconference](http://unconf18.ropensci.org) in May 2018, a group of scientists and educators shared their experiences, frustrations, failures, and successes teaching science with R. What came out of this discussion was a framework for rOpenSci Educators’ Collaborative: a community of practice for people interested and engaged in science education using R. This blog post is the first of a three-post series about education and R, written by this group of unconf18 participants. Read about how to get involved [here](https://github.com/ropenscilabs/rOpenSciEd)."
+authors:
+  - name: Laura Ación
+    url: https://github.com/lauracion
+  - name: Mara Averick 
+    url: https://github.com/batpigandme
+  - name: Leonardo Collado Torres
+    url: https://lcolladotor.github.io
+  - name: Auriel Fournier 
+    url: https://github.com/aurielfournier
+  - name: Alison Hill
+    url: https://alison.rbind.io
+  - name: Sean Kross
+    url: https://github.com/seankross
+  - name: Lincoln Mullen
+    url: https://lincolnmullen.com
+categories: blog
+topicid: 
+tags:
+- R
+- community
+- meetings
+- unconf
+- unconf18
+- educollab
```

Comment:
checked twitter #educollab gets used a little bit: https://twitter.com/search?f=tweets&vertical=default&q=%23educollab&src=typd

There's also this: https://outreach.wikimedia.org/wiki/Education/Wikipedia_Education_Collaborative#Current_members

I like the sound of educollab. Don't know how far you want to go to have a unique tag

Reply:
Let use educollab then.